oak_containers/proto/launcher.proto

```diff
@@ -27,11 +27,20 @@ message GetImageResponse {
   bytes image_chunk = 1;
 }
 
+message GetContainerConfigResponse {
+  // Arbitrary config that the container can retrieve from the orchestrator.
+  // Included in the attestation measurements conducted by the orchestrator.
+  bytes config = 1;
```

Comment:
Could you provide some more information about the type of config you see this being used for?

Reply:
This could be anything that the application wants to attest to. e.g. whether it's in debug mode, or parameters for aggregation / DP, or digest of something else that the application will load later (e.g. Wasm module). It may be empty if the application does not require one